src/common/enums.ts

```diff
@@ -4,4 +4,11 @@ enum ArtifactType {
   GPKG = 'GPKG',
 }
 
+export enum TileFormatStrategy {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  Fixed = 'fixed',
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  Mixed = 'mixed',
+}
```

Comment:
Why are you disabling `eslint` on these lines?

Reply:
Because lint expects CAPITAL leters in enum values

Reply:
And what's the problem to make it CAPITAL?

Reply:
As i know, its should be CAPITAL , ex. TileFormatStrategy.FIXED

Reply:
because the merger expects it to be lower case, do you want me to change to CAPITAL and add to lower when putting in task parameters?